docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+---
+toc_max_heading_level: 5
+---
+
+# Build
+
+The Unity orb provides the
+[build job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-build) to facilitate the
+process of building your Unity project to multiple platforms.
+
+### Parameters
+
+In this sub-section, you will find information on each parameter the job provides.
+
+#### build-name
+
+- _Default_: `UnityBuild`
+- _Required_: `False`
+- _Type_: `String`
+
+Your build's name. If you choose to compress your artifacts, this name will be used in the
+compressed file.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          build-name: 'MyUnityProject'
+```
+
+#### build-target
+
+- _Default_: `N/A`
+- _Required_: `True`
+- _Type_: `String`
+
+The platform on which you want to run your game. Valid parameters can be found in the Unity Editor
+[documentation](https://docs.unity3d.com/2022.2/Documentation/ScriptReference/BuildTarget.html).
```

Comment:
```suggestion
The platform on which you want to run your game or application. Available targets can be found in the Unity Editor
[documentation](https://docs.unity3d.com/2022.2/Documentation/ScriptReference/BuildTarget.html).
```

Reply:
Quick note: even though we're called GameCI, we acknowledge that Unity is used for many use cases outside of games, like educational projects, 3d guided tours and many more.

Reply:
That's fair. And done ✔️